<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTerrainNodePage</class>
 <widget class="QWidget" name="QTerrainNodePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>5</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="m_name"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Material:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="MaterialComboBox" name="m_material"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Height Map:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="TextureComboBox" name="m_heightMap"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MaterialComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">SceneEditorCore/MaterialComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>TextureComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">SceneEditorCore/TextureComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
